llms/mlx_lm/lora.py

```diff
@@ -43,6 +43,26 @@
     "model": "mlx_model",
     "train": False,
     "fine_tune_type": "lora",
+    "optimizer": "adam",
+    "optimizer_config": {
+        "adam": {
+            "betas": [0.9, 0.999],
+            "eps": 1e-8,
+            "bias_correction": False
+        },
+        "adamw": {
+            "betas": [0.9, 0.999],
+            "eps": 1e-8,
+            "weight_decay": 0.01,
+            "bias_correction": False
+        },
+        "muon": {
+            "momentum": 0.95,
+            "weight_decay": 0.01,
+            "nesterov": True,
+            "ns_steps": 5
+        }
```

Comment:
Get rid of `muon` since it's not in the core package. We can add it back in if that changes.